docs/contacts/preference_center.rst

```diff
@@ -4,5 +4,192 @@ Preference center
+The second tab gives the option to add or remove the Contact from global Categories used in Emails or Categories. 
+The third tab allowS to add or remove the Contact from Segments it belongs to.
+
+.. vale off
+
+Preferred Channels and frequency
+********************************
+
+.. vale on
+
+.. image:: images/preferences.png
+    :align: center
+    :alt: Screenshot of Preference
+
+|
+
+In this window you can switch Channels of communication, set the frequency of the communication via each Channel enabled, and set one of the Channels as a preferred Channel.
+
+To set a Channel as a don't contact me through this Channel, un-tick the tick-box next to the Channel name in the first column.
+
+When a Channel is selected, these will be used to send marketing messages if there is a message set for any of the Channels selected. You can also set the frequency of the communication, as in this example the frequency is set to "Send me emails twice a day" but pause them during the "``1st`` of November 2022 to the ``30th`` of November 2022" . Email is also set as the preferred Channel, so if the same message is set for both email and ``sms``, it will only send the Email version of the message to the selected Contact.
+
+.. vale off
+
+Contact Categories
+******************
+
+.. vale on
+
+.. image:: images/categories.png
+    :align: center
+    :alt: Screenshot of Categories
+
+|
+
+Use the Categories tab to add or remove a Contact from a global Category. Global Categories can be used in areas like Emails, text messages, Campaigns. In combination with the new Subscribed Categories Segment filter, Contacts can be given the choice to opt out of categorized communications.
+
+.. vale off
+
+Contact Segments
+****************
+
+.. vale on
+
+.. image:: images/segments.png
+    :align: center
+    :alt: Screenshot of Segments
+
+|
+
+Use the Segments tab to add or remove a Contact from a Segment. Segments are used as a source for Campaigns and Emails. Any Contact in a particular Segment will be part of a Campaign that has that Segment as the source. You can also use a standalone Email manually to a Segment. If a User has opted out of a Segment it will no longer receive Campaign actions or messages sent to that Segment.
+
+.. vale off
+
+Contact's Unsubscribe Email Preferences
+***************************************
+
+.. vale on
+
+.. image:: images/email-unsubscribe-settings.png
+    :align: center
+    :alt: Screenshot of Email unsubscribe
+
+|
+
+The Contact's preferences can be presented to the User in the unsubscribe page by selecting "Show Contact preference settings" in the Email configuration. You may also choose to hide or show different Segments of the User preferences. If any of these areas is set to no, it hides it from the contact's personal preferences page. The default unsubscribe message is shown if the preference setting option is set to no.
```

Comment:
Might be worth linking this section to the part in the global configuration about the email preference centre options?